<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="screamerVerticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLineEdit" name="screamerLine"/>
      <widget class="QSplitter" name="screamerButtonSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushStandartValueButton">
        <property name="text">
         <string>StandartValue</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delFromScreamerButton">
        <property name="text">
         <string>delFromScreamer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addToScreamerButton">
        <property name="text">
         <string>addToScreamer</string>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="screamerListWidget"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="observerVerticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLineEdit" name="observerLine">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="observerButtonSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="addObserverButton">
        <property name="text">
         <string>addObserver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delObserverButton">
        <property name="text">
         <string>delObserver</string>
        </property>
       </widget>
       <widget class="QPushButton" name="renameObserverButton">
        <property name="text">
         <string>renameObserver</string>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="observerDisplayWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushStandartValueButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pushStandartValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delFromScreamerButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delFromScreamer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addToScreamerButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addToScreamer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addObserverButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addObserver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delObserverButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delObserver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renameObserverButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>renameObserver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateScreamerDisplayWidget()</slot>
  <slot>addToScreamer()</slot>
  <slot>delFromScreamer()</slot>
  <slot>addObserver()</slot>
  <slot>delObserver()</slot>
  <slot>renameObserver()</slot>
  <slot>pushStandartValue()</slot>
 </slots>
</ui>
